test/parallel/test-stream-writable-invalid-chunk.js

```diff
@@ -2,20 +2,19 @@
 
 const common = require('../common');
 const stream = require('stream');
+const assert = require('assert');
 
 function testWriteType(val, objectMode, code) {
   const writable = new stream.Writable({
     objectMode,
     write: () => {}
   });
-  if (!code) {
-    writable.on('error', common.mustNotCall());
-  } else {
-    writable.on('error', common.expectsError({
-      code: code,
-    }));
+  writable.on('error', common.mustNotCall());
+  if (code) {
```

Comment:
We should test that no error is thrown when code is falsy no?

Reply:
If an error is thrown won't it crash it?

Reply:
Ah, I missed that. Thanks!